components/Chart.tsx

```diff
@@ -239,7 +239,7 @@ const Chart = (props: Props) => {
       patternClassName: _.find(patternClasses, ['_id', term.pattern?.class?._ref])?.name,
       patternClassOrder: _.find(patternClasses, ['_id', term.pattern?.class?._ref])?.order,
       patternIconUrl: term.pattern?.iconUrl,
-      type: term.type,
+      type: term.type === "Limitation" ? "Obligation" : term.type,
```

Comment:
:eyes: definitely hadn't realized we had a distinct "Limitation" type, merging with "Obligation" for now seems right to me (and aiming to retire/remove from Sanity longer term?)

Reply:
Yeah - we have talked about "Limitation" before but I'm a bit hazy on it - I think it's just an alternate wording. This works for now, and we can tidy up Sanity later 👍